tensorflow/java/README.md

```diff
@@ -85,6 +85,8 @@ Java bindings for TensorFlow. ([Javadoc](https://www.tensorflow.org/api_docs/jav
       org.tensorflow.examples.LabelImage
     ```
 
+    Note: you can also dynamically link the native library at runtime by using `System.load()`.
```

Comment:
I'd suggest removing this. It's a hard line to draw between keeping the README short and focused enough for the common use case and covering all possibilities. In this particular case, I'd prefer omitting this.